.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### Short description
+I have:
+- [ ] There is a corresponding issue for this pull request.
+- [ ] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
+
+### For the reviewers
+I have:
+- [ ] Reviewed this pull request by an authorized contributor.
```

Comment:
Yea I know that, that's why I suggested to use:
>Reviewed this pull request as an authorized 
                                                  ^^
contributor.

Instead of:
>Reviewed this pull request by an authorized 
                                                  ^^ 
contributor.